docs/agile-process.md

```diff
@@ -0,0 +1,30 @@
+FAC - Agile Process
+===================
+
+### Overview
+
+There is a [high-level roadmap](https://app.mural.co/t/gsa6/m/gsa6/1646336450605/6a77a27a287989f91e1c0dfbcf9be4dec6e49393?sender=ua4d37dfba3f1e69e09078790) that the product and design team distills down to actionable tasks at the beginning of each sprint. Design, engineering and product work is tracked on the [FAC Task Tracking board](https://github.com/GSA-TTS/FAC/projects/1) in Github. "Story" issues can be created to define a high-level piece of functionality, but linked subtasks should be created for discrete pieces of work across disciplines, each not exceeding one day in anticipated time. There is no estimation process for tickets, but they should be groomed to be in approximately comparable sizes. The team works in two-week sprints starting on Tuesdays.
+
+### Board workflow
+
+-   **Backlog** - Not currently selected for the current sprint
+
+-   **Available** - Selected for the current sprint and is currently available to be picked up.
```

Comment:
Suggesting we rename “To do” to “Available” to communicate these issues can be picked up